src/app/core/layers/providers/qgisprovider.js

```diff
@@ -455,7 +457,7 @@ proto._loadVectorData = function(vectorLayer, bbox) {
       vectorLayer.setData(vectorDataResponse.vector.data);
       if (this._) return vectorDataResponse;
     })
-    .fail(() => {
+    .catch(() => {
       return false;
     })
```

Comment:
double check, the following statement can be quite ambiguous:

```js
.catch(() => { return false; })
```

it is best to make explicit what the intent is (ie. `reject` or `resolve` the promise) by using one of the following statements:

```js
.catch(() => resolve(false))
```
```js
.catch(() => reject(false))
```

Reply:
@Raruto you can replace .catch(() => { return false; }) with .catch(() => reject(false)). We double check if this method is still used.